src/components/forms/Radio/Radio.stories.tsx

```diff
@@ -33,3 +33,26 @@ export const selected = (): React.ReactElement => (
 export const disabled = (): React.ReactElement => (
   <Radio id="input-radio" name="input-radio" label="My Radio Button" disabled />
 )
+
+export const WithLabelDescription = (): React.ReactElement => (
+  <Radio
+    id="input-radio"
+    name="input-radio"
+    label="My Radio Button"
+    labelDescription="Optional label description"
```

Comment:
Same comment as the other feedback: https://github.com/trussworks/react-uswds/pull/1104#discussion_r612470944

Reply:
changed